pixi/src/ui/recommendations/RecommendationsView.js

```diff
@@ -83,14 +83,15 @@ export default class RecommendationsView {
         recommendation.classList.add(tagId);
         tag.textContent = i18n.getText(`tags.${tagId}`);
         tagsBar.appendChild(tag);
-        tagIds.push(tagId);
+        const count = tagIdCounts[tagId] || 0;
+        tagIdCounts[tagId] = count + 1;
       }
 
       this.recommendationNodes.push(recommendation);
       this.container.appendChild(recommendation);
     }
 
-    for (const tagId of new Set(tagIds).values()) {
+    for (const tagId of Object.keys(tagIdCounts)) {
```

Comment:
why this change?

Reply:
The Set was used to get unique values from an array with duplicates. I replaced the array with duplicates with an object that keeps count of values instead.